src/js/load-package.ts

```diff
@@ -165,7 +192,14 @@ class PackageManager {
     const loadedPackageData = new Set<InternalPackageData>();
     const messageCallback = options.messageCallback || console.log;
     const errorCallback = options.errorCallback || console.error;
-    if (names instanceof PyProxy) {
+
+    // originally, this condition was "names instanceof PyProxy",
+    // but it is changed to check names.toJs so that we can use type-only import for PyProxy and remove side effects.
+    // this change is required to run unit tests against this file, when global API or Module is not available.
+    // TODO: remove side effects from pyproxy.ts so that we can directly import PyProxy
+    // @ts-ignore
+    if (typeof names.toJs === "function") {
+      // @ts-ignore
       names = names.toJs();
     }
     if (!Array.isArray(names)) {
```

Comment:
I wonder if we should check here if `typeof names  === "string"` and then we wouldn't need the type assertion.

Reply:
Thanks for the suggestion. I tried that change but TS was complaining about the PyProxy type as the type is not narrowed enough.